src/controllers/index.js

```diff
@@ -15,11 +16,19 @@ router.get('/notmember', (req, res) => {
 
 // PROTECTED ROUTES //
 router.get('/myprofile/:github_id', ensureAuthenticated, profile.get);
-router.post('/saveDetails', ensureAuthenticated, (req, res) => {
+router.post('/savePersonalDetails', ensureAuthenticated, (req, res) => {
   // post user data (req.body) to database
   console.log('form data: ', req.body);
   res.redirect('/myprofile/:github_id');
 });
+router.post('/saveJobDetails', ensureAuthenticated, (req, res) => {
+  console.log('form data: ', req.body);
+
+  return saveJobDetails(req.body, req.user.github_id).then(() => {
+    res.redirect('/myprofile/:github_id');
+  })
+    .catch(err => console.log('Error saving job details: ', err));
```

Comment:
We should give the client feedback when something has gone wrong (related to Elias' code review on dealing with errors  #86)